### PR TITLE
Fix login dialog causing main window to be non movable

### LINKED ISSUE #2378: Cannot move window via top bar or resize window after clicking the add account button
**Describe the bug**
After clicking the add account button, you cannot move the window using the top bar or resize using the edge of the application

**To reproduce**
1. Click and drag the window using the top bar, notice it works normally
2. Go to settings to add account, click add account
3. Close the window, close the settings

**Screenshots**

https://user-images.githubusercontent.com/41973452/160349134-e34508d1-20dd-4550-8212-b6bf956e0f43.mp4

**Chatterino version**
commit a9b1af60d

**Operating system**
Windows 10

### PR DESCRIPTION
**Pull request by kornes**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
fixes #2378, started in #3636

also added missing parent to messageBox and changed buttons to be in 2 rows to better use empty space

screen
I feel like "step" labels might help some users better understand login flow, but wont mind removing them

### CODE REVIEW

**Comment by Felanbird** [comment]
Issue seems to be resolved, even with `Always on Top` enabled, but I'm sure it would be preferred if the step labels were added in a second PR

**Comment by kornes** [comment]
reverted buttons layout/labels change

File: src/widgets/dialogs/LoginDialog.cpp [hunk 1]
```diff
@@ -84,7 +77,7 @@ BasicLoginWidget::BasicLoginWidget()
     const QString logInLink = "https://chatterino.com/client_login";
     this->setLayout(&this->ui_.layout);
 
-    this->ui_.loginButton.setText("Log in (Opens in browser)");
+    this->ui_.loginButton.setText("Login using browser");
```
Comment (Felanbird) [inline src/widgets/dialogs/LoginDialog.cpp:80]: This in 2nd PR as well

File: src/widgets/dialogs/LoginDialog.cpp [hunk 2]
```diff
@@ -238,15 +233,15 @@ void AdvancedLoginWidget::refreshButtons()
...
-    ::SetWindowPos(HWND(this->winId()), HWND_TOPMOST, 0, 0, 0, 0,
-                   SWP_NOMOVE | SWP_NOSIZE | SWP_SHOWWINDOW);
-#endif
+    this->setMinimumWidth(300);
+    this->setWindowFlags(
+        (this->windowFlags() & ~(Qt::WindowContextHelpButtonHint)) |
+        Qt::Dialog | Qt::MSWindowsFixedSizeDialogHint);
 
-    this->setWindowTitle("Chatterino - add new account");
+    this->setWindowTitle("Add new account");
```
Comment (Felanbird) [inline src/widgets/dialogs/LoginDialog.cpp:244]: This in 2nd PR as well

**Review by Felanbird — DISMISSED** [review]

File: src/widgets/dialogs/LoginDialog.cpp (hunk 2 above)
Comment (kornes) [reply src/widgets/dialogs/LoginDialog.cpp:244]: this change is in scope - window now acts as modal dialog, these should only use feature name in caption when initiated by user action

File: src/widgets/dialogs/LoginDialog.cpp (hunk 1 above)
Comment (kornes) [reply src/widgets/dialogs/LoginDialog.cpp]: done

**Review by pajlada — APPROVED** [review]

**Pull request merged**